converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/FhirTransactionPipeline.cs

```diff
@@ -86,11 +97,23 @@ public async Task ProcessAsync(ChangeFeedEntry changeFeedEntry, CancellationToke
             Bundle responseBundle = await _fhirTransactionExecutor.ExecuteTransactionAsync(bundle, cancellationToken);
 
             // Process the response.
+            int processesResponseItems = 0;
```

Comment:
Can we use `responseEntries.Count` instead?

Reply:
We're using this as an offset of the already processed accessors in the FHIR response bundle since they all come back in order. We need to grab the next unprocessed one.